common/src/main/java/com/microsoft/identity/common/internal/providers/microsoft/azureactivedirectory/ClientInfo.java

```diff
@@ -40,7 +42,7 @@ public ClientInfo(String rawClientInfo) {
         try {
             clientInfoItems = JsonExtensions.extractJsonObjectIntoMap(decodedClientInfo);
         } catch (final JSONException e) {
-            throw new RuntimeException(e);
+            throw new CommonCoreServiceException("", CommonCoreExceptionMessage.INVALID_JWT, e);
```

Comment:
Here's an example that i'd like us to review.  Originally this was a client exception in MSAL.  Is it our expectation that malformed JSON returned from the server would be correct on a subsequent request?  How does the developer know that an invalid JWT is something that they can recover from?  Do they need to make a new silent or a new interactive request?